package.json

```diff
@@ -58,6 +58,7 @@
     "@dbeining/react-atom": "^4.1.21",
     "@jest/globals": "^29.5.0",
     "@libre/atom": "^1.3.3",
+    "@types/inquirer": "^9.0.7",
```

Comment:
think we don't need this. can instead just use the `prompts` we already have a dep for